build-go/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/asraa/slsa-on-github/build-go
+
+go 1.17
+
+require gopkg.in/yaml.v2 v2.4.0
```

Comment:
really small nit but i'd update this
https://github.com/advisories/GHSA-wxc4-f4m6-wwqv

Reply:
the link says versions < 2.2.8 are vulnerable. So 2.4 is fine, right?